subsys/net/lib/nrf_cloud/src/nrf_cloud_codec_internal.c

```diff
@@ -1694,6 +1733,18 @@ int nrf_cloud_shadow_dev_status_encode(const struct nrf_cloud_device_status *con
 		parent_obj = cJSON_AddObjectToObjectCS(root_obj, NRF_CLOUD_JSON_KEY_REP);
 	}
 
+	cJSON *connection_obj = cJSON_AddObjectToObjectCS(parent_obj, NRF_CLOUD_JSON_KEY_CONN);
```

Comment:
i realize this is supposed to be temporary, but who knows how long it will remain...
Can you add a kconfig to disable adding the new connection data?
All the other device status stuff can be turned on/off via the `nrf_cloud_device_status` struct.

Reply:
Correct, method will need to always be device-reported.